bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/GiniBank.kt

```diff
@@ -628,7 +628,7 @@ object GiniBank {
 
     /**
      *
-     *  Kills Bank sdk and stos Gini payment flow
+     *  Stops Bank sdk and stos Gini payment flow
```

Comment:
I think this typo is still there which was pointed out by lenci, which is "stos" in this line "Stops Bank sdk and this word -> stos Gini payment flow"